pages/cloudflare/cli.mdx

```diff
@@ -0,0 +1,35 @@
+import { Callout } from "nextra/components";
+
+## `opennextjs-cloudflare` CLI
+
+The cloudflare adapter provides a `opennextjs-cloudflare` CLI to develop, build, and deploy your application. You should not use `wrangler` commands directly unless documented otherwise or if you know what you are doing.
```

Comment:
```suggestion
The Cloudflare adapter provides a `opennextjs-cloudflare` CLI to develop, build, and deploy your application. You should not use `wrangler` commands directly unless documented otherwise or if you know what you are doing.
```